### PR TITLE
Configuration Section Setup from Sidebar

### PR DESCRIPTION
**Pull request by Vinyl-Davyl**
Setup from Configuration section

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import { Routes, Route } from "react-router-dom";
 import { ThemeProvider } from "@mui/material";
-import { DevDashboard, Home, Login, Signup, UserProfile, ForgotPassword, LoginHistory, Otp, APIPage, Analytics, SuccessPage, Configuration } from "./pages";
+import { DevDashboard, Home, Login, Signup, UserProfile, ForgotPassword, LoginHistory, Otp, APIPage, Analytics, SuccessPage } from "./pages";
```
Comment (deviator53) [inline src/App.tsx]: @Vinyl-Davyl Configuration is not imported

Comment (Vinyl-Davyl) [reply src/App.tsx]: No, Its all good now! I remodified it and I'm no longer using configuration in the App.tsx

Comment (Vinyl-Davyl) [reply src/App.tsx]: pls confirm It again

**Review by deviator53 — APPROVED** [review]

File: src/components/AddApi.tsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import { makeStyles } from "@mui/styles";
 import FileUploadIcon from '@mui/icons-material/FileUpload';
```
Comment (pablo-clueless) [inline src/components/AddApi.tsx:3]: Why absolute imports?? We agreed on named imports unless not available

File: src/components/ApiPageSidebar.tsx [hunk 3]
```diff
@@ -13,6 +14,7 @@ interface TabPanelProps {
 }
 type Props = {
   addApi: ReactNode,
+  configuration: ReactNode,
 };
 function TabPanel(props: TabPanelProps) {
```
Comment (pablo-clueless) [inline src/components/ApiPageSidebar.tsx]: Use ES6, please.

File: src/components/AddApi.tsx (hunk 2 above)
Comment (Vinyl-Davyl) [reply src/components/AddApi.tsx:3]: @Yagazie-davidson Should answer this.
I'm just basically working on a subsection in this file

**Review by taiwoAkindele — APPROVED** [review]

File: src/components/AddApi.tsx (hunk 2 above)
Comment (Vinyl-Davyl) [reply src/components/AddApi.tsx:3]: Resolved

File: src/components/ApiPageSidebar.tsx (hunk 3 above)
Comment (Vinyl-Davyl) [reply src/components/ApiPageSidebar.tsx]: Resolved

**Pull request merged**